boards/arm/rak4631_nrf52840/rak4631_nrf52840.dts

```diff
@@ -110,7 +110,7 @@
 		busy-gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
 		tx-enable-gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
 		rx-enable-gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
-		dio1-gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		dio1-gpios = <&gpio1 15 (GPIO_PULL_DOWN | GPIO_ACTIVE_HIGH)>;
```

Comment:
I don't believe that the `GPIO_PULL_DOWN ` flag is needed. I have multiple boards using this IC that don't require it, and the datasheet itself specifies that this pin is either actively driven or has an internal pull down in all modes.
![image](https://user-images.githubusercontent.com/8476715/185070955-35fa3efd-6483-40fb-b055-2b5d4017ccff.png)

Reply:
Thanks for the hint, done some tests here without PullDown and everything seems alright. Changed the .dts file and commited again for your review.